src/main/java/frc/robot/constants/HardwareConstants.java

```diff
@@ -109,9 +109,9 @@ public record ArmConstants(
             String CANBus,
             int leftPivotMotorId,
             int rightPivotMotorId,
-            int pivotUpperLimitSwitchDIOChannel,
```

Comment:
Did we already physically remove the magnetic limit switch? If not, I don't see a reason we couldn't keep it in code in case we need to zero something using it.

Reply:
Yes we did. They also covered the DIO ports.

Reply:
Ok, just checking. Would've been nice if we didn't 😞.